script.js

```diff
@@ -1,4 +1,176 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (
+    input != "scissors" ||
+    input != "paper" ||
+    input != "rock" ||
+    input != "reversed scissors" ||
+    input != "reversed paper" ||
+    input != "reversed rock"
+  ) {
+    myOutputValue =
+      "Please try again and input the following : Scissors, Paper, Rock, Reversed Scissors, Reversed Paper, Reversed Rock";
```

Comment:
nice to check input immediately!

you can consider adding return to line 10 to immediately stop the program running the rest of the code when you are not getting the input you want